content/2017/10/maxmind-will-be-exhibiting-at-ad-tech-november-1-and-2.mdx

```diff
@@ -2,8 +2,8 @@
 title: "MaxMind Will Be Exhibiting at ad:tech November 1 and 2"
 date: "2017-10-09"
 categories:
-  - "category-ip-geolocation"
-  - "online-fraud-prevention"
+  - "IP Geolocation and Network Data"
```

Comment:
@christophermluna Can you please confirm whether these updated category names are correct?

Reply:
this looks right, thanks